src/threshold-ts/utils/math.ts

```diff
@@ -0,0 +1,9 @@
+import { BigNumber } from "ethers"
+
+export const min = (a: string | number, b: string | number) => {
+  return BigNumber.from(a).lt(b) ? a.toString() : b.toString()
+}
```

Comment:
I think it would be better to return the value with the same type as it was passed. This will make it more reusable:

```suggestion
import { BigNumber, BigNumberish } from "ethers"

export const min = (a: BigNumberish, b: BigNumberish) => {
  return BigNumber.from(a).lt(b) ? a : b
}
```

What do you think? (P.S I did not test it)

Reply:
Hmm hard to say because you can pass `a` as `number` and `b` as `string`- in that case, function returns `a` as `number` but `b` as `string`. Maybe let's convert result to `BigNumber`?